src/components/thread/ThreadSidebar.tsx

```diff
@@ -57,24 +67,24 @@ const ThreadSidebar: React.FC<ThreadSidebarProps> = (props) => {
                 id: THREAD_VIEW_MODE.THREAD,
                 label: "Thread",
                 link: {
-                  // TODO: add href here
                   onClick: () => props.onViewChange(THREAD_VIEW_MODE.THREAD),
+                  href: `${linkToThread.href}?thread`
```

Comment:
I wasn't sure how to set the view in the URL, since `linkToThread` gets generated before this.

Reply:
Just generate multiple of theand put them here:

```ts
  const linkToGallery = getLinkToThread({
    slug,
    threadId,
    view: "gallery"
  }).href;
const linkToTimeline = getLinkToThread({
    slug,
    threadId,
    view: "timeline"
  }).href;
```